microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -904,30 +948,98 @@ See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 125.
-* This checkbox is present in Microsoft Edge Stable 124.
+* This checkbox is not present in Microsoft Edge Canary 128.
+* This checkbox is present in Microsoft Edge Stable 125.
+
+
+<!-- ====================================================================== -->
+## Performance panel: enable user timings based extensions
+<!-- checkbox has no (?) link pointing to this anchor wording -->
+
+_new label for checkbox as of Microsoft Edge 128_
+
+Enables extensions that customize the appearance of user timing events in the **Performance** tool.
+
+See also:
+* [Performance features reference](../evaluate-performance/reference.md)
+* [Performance recording event reference](../evaluate-performance/performance-reference.md)
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 128.
+* This checkbox is present in Microsoft Edge Stable 125.
 
 
 <!-- ====================================================================== -->
 ## Timeline: Enable User Timings based extensions
-<!-- no (?) link -->
+<!-- checkbox has no (?) link pointing to this anchor wording -->
 
 _new as of Microsoft Edge 125_
 
-Enables extensions that customize the appearance of user timing events in the **Performance** tool.
+See the above section, showing the newer label for the checkbox.
+
+
+<!-- ====================================================================== -->
+## Performance panel: Enable collecting enhanced traces
+<!-- checkbox has no (?) link pointing to this anchor wording -->
+
+_new as of Microsoft Edge 128_
+
+In the **Performance** tool, when recording, collects enhanced traces.
+<!-- todo: real desc - maybe extract from article -->
```

Comment:
Performance panel: Enable collecting enhanced traces
Need real desc @captainbrosset
try from article:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/share-traces?branch=pr-en-us-3192

Reply:
@robpaveza can you help with this one please? Isn't this something that Edge has been able to do for some time, and we're upstreaming it now. Trying to understand whether enabling this experiment downstream does anything, or whether we just always support enhanced traces.

Reply:
Jayson to assist

Reply:
This experiment is for performance panel capturing newly added traces to support enhanced traces v2 (currently implementing in upstream). It does not affect the ability of downstream enhanced traces and the state of it doesn't impact downstream user's experience.

Reply:
result:
```
This experiment makes the **Performance** tool capture newly added traces, to support 
enhanced traces v2.  This feature is being implemented in upstream.  This experiment 
doesn't affect the capabilities of downstream enhanced traces.  The state of this checkbox 
doesn't impact a downstream user's experience.
```